notebooks/pixiedust_facebook_analysis.ipynb

```diff
@@ -1073,28 +1073,12 @@
    },
    "outputs": [],
    "source": [
-    "def put_file(credentials, local_file_name):\n",
-    "    \"\"\"Put the file in IBM Object Storage.\"\"\"\n",
-    "    f = open(local_file_name,'r',encoding=\"utf-8\")\n",
-    "    my_data = f.read()\n",
-    "    data_to_send = my_data.encode(\"utf-8\")\n",
-    "    url1 = ''.join(['https://identity.open.softlayer.com', '/v3/auth/tokens'])\n",
-    "    data = {'auth': {'identity': {'methods': ['password'],\n",
-    "            'password': {'user': {'name': credentials['username'],'domain': {'id': credentials['domain_id']},\n",
-    "            'password': credentials['password']}}}}}\n",
-    "    headers1 = {'Content-Type': 'application/json'}\n",
-    "    resp1 = requests.post(url=url1, data=json.dumps(data), headers=headers1)\n",
-    "    resp1_body = resp1.json()\n",
-    "    for e1 in resp1_body['token']['catalog']:\n",
-    "        if(e1['type']=='object-store'):\n",
-    "            for e2 in e1['endpoints']:\n",
-    "                        if(e2['interface']=='public'and e2['region']=='dallas'):\n",
-    "                            url2 = ''.join([e2['url'],'/', credentials['container'], '/', local_file_name])\n",
-    "    s_subject_token = resp1.headers['x-subject-token']\n",
-    "    headers2 = {'X-Auth-Token': s_subject_token, 'accept': 'application/json'}\n",
-    "    resp2 = requests.put(url=url2, headers=headers2, data = data_to_send )\n",
-    "    resp2.raise_for_status()\n",
-    "    print('%s saved to IBM Object Storage. Status code=%s.' % (localfilename, resp2.status_code))"
+    "cos = ibm_boto3.client(service_name='s3',\n",
+    "    ibm_api_key_id=credentials['IBM_API_KEY_ID'],\n",
+    "    ibm_service_instance_id=credentials['IAM_SERVICE_ID'],\n",
```

Comment:
is IAM supposed to be IBM?   I think I've seen the IAM abbrev for an auth thing, but looks like a typo here.

Reply:
no, IAM_SERVICE_ID is correct,
and the notebook is tested and works.

Reply:
ok dokey.  Let's guess Identity Auth... Management?